server/lorax_server/models/__init__.py

```diff
@@ -92,6 +92,10 @@ def get_model(
             dtypetrust_remote_code=trust_remote_code,
         )
 
+    if "WhereIsAI/UAE-Large-V1" in model_id:
```

Comment:
Is there a way we can make this more generic to support any BERT based model?

Reply:
Good call out